packages/expo-clipboard/src/Clipboard.types.ts

```diff
@@ -78,3 +80,81 @@ export interface SetStringOptions {
    */
   inputFormat?: StringFormat;
 }
+
+// @needsAudit
+export interface ClipboardPasteButtonProps extends ViewProps {
+  /**
+   * A callback that is called with the result of the paste action.
+   * Inspect the `type` property to determine the type of the pasted data.
+   
+   * Can be one of `text` or `image`.
+   *
+   * @example
+   * ```ts
+   *   onPress={(data) => {
+   *     if (data.type === 'image') {
+   *       setImageData(data);
+   *    } else {
+   *       setTextData(data);
+   *     }
+   *   }}
+   * ```
+   */
+  onPress: (data: PasteEventPayload) => void;
+  /**
+   * The backgroundColor of the button.
+   */
+  backgroundColor?: string;
+  /**
+   * The foregroundColor of the button.
+   */
+  foregroundColor?: string;
```

Comment:
Consider allowing `null` values in optional fields.

Also, if possible, let's document what are the default values. We can even say that the platform automatically chooses the default value based on the system settings (if that's true).